Source/drlg_l1.cpp

```diff
@@ -741,18 +741,17 @@ void ApplyShadowsPatterns()
 	}
 }
 
-bool PlaceMiniSet(const Miniset &miniset, bool setview)
+bool PlaceMiniSet(const Miniset &miniset, Point *position = nullptr)

```

Comment:
Returning `optional<Point>` was the approach I was considering for this earlier. You can use the return value as a bool with the existence check instead of using a parameter as a conditional out.